content/en/getting-started/api-key.md

```diff
@@ -0,0 +1,85 @@
+---
+title: "API Key"
+weight: 5
+categories: ["LocalStack Pro & Enterprise"]
+description: >
+  Configure your API key to start LocalStack
+---
+
+The LocalStack API key is a unique identifier to authenticate your LocalStack instance. You can find your API key in the [LocalStack dashboard](https://app.localstack.cloud/account/apikeys). This guide demonstrates how you can use your new LocalStack licenses and go over some best practices regarding the usage, activation, and safety of your LocalStack API key.
```

Comment:
nit: We could try to be consistent with the naming - _LocalStack Web app_ seems to be the name we've been using most frequently thus far.
```suggestion
The LocalStack API key is a unique identifier to authenticate your LocalStack instance. You can find your API key in the [LocalStack Web app](https://app.localstack.cloud/account/apikeys). This guide demonstrates how you can use your new LocalStack licenses and go over some best practices regarding the usage, activation, and safety of your LocalStack API key.
```